pmm-app/src/pmm-dbaas/components/DBCluster/AddDBClusterModal/DBClusterAdvancedOptions/DBClusterAdvancedOptions.tsx

```diff
@@ -112,6 +117,13 @@ export const DBClusterAdvancedOptions: FC<FormRenderProps> = ({
           disabled={resources !== DBClusterResources.custom}
           parse={parsePositiveInt}
         />
+        <NumberInputField
```

Comment:
Please add tests to DBClusterAdvancedOptions.test to validate this new field

Reply:
forgot about tests, thanks, added

Reply:
Good job!!